### PR TITLE
Simplified Chinese README

### PR DESCRIPTION
**Pull request by clarkzjw**
I translated the README file to simplified Chinese :smile: 
Just a little bit of contribution.

### CODE REVIEW

**Comment by callumacrae** [comment]
I mean I don't understand Chinese, but this looks good to me!

**Comment by alfredxing** [comment]
Thanks! :+1:
From what I can understand, this is a direct translation from [the 正體 version](https://github.com/muan/hello-world/blob/gh-pages/README-zhtw.md)?

**Comment by clarkzjw** [comment]
@alfredxing Well, actually, the differences between traditional Chinese and simplified Chinese are quite small, so sometimes a direct translation is just ok.

But, for some historical reasons, there do exists differences in some words or grammar preferences between traditional Chinese and simplified Chinese.To give an example, "Computer programs" are translated into "程序" in simplified Chinese, but "程式" in traditional Chinese.

**Comment by alfredxing** [comment]
@JinweiClarkChao Yep. But the content is basically the same, right?

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ If you're looking for resources to learn about web programming, checkout @jlord'
 
 [`正體中文的介紹檔案`](https://github.com/muan/hello-world/blob/gh-pages/README-zhtw.md)
 
+[`简体中文的介绍文件`](README-zhcn.md)
```
Comment (muan) [inline README.md:11]: let's make the translations a list :), i.e. 
- first lang
- second lang

**Comment by clarkzjw** [comment]
@alfredxing yes

**Comment by clarkzjw** [comment]
@alfredxing Mostly, the characters used in traditional Chinese and simplified Chinese are exactly the same .

**Comment by muan** [comment]
Thanks! I'll merge this now.

**Pull request merged**